### ISSUE TITLE
add ability to confirm that the pop node is running after `pop up parachain` command

### ISSUE THREAD

**Issue by brunopgalvao** [r0gue-io/pop-cli#117]
As a user, I would like for a browser window to open with Pop Network upon successful completion of the command:
```
pop up parachain -f ./network.toml -p https://github.com/r0gue-io/pop-node                                                                                                                    
```
So that I do not have to manually open a browser window to confirm that Pop Network is running.

> Either that or open a terminal with the Pop Network logs running live `tail -f pop.log`

**Comment by chungquantin** [comment]
Is this open for public contributors? I would love to work on this enhancement

**Comment by AlexD10S** [comment]
Absolutely, go ahead! We're thrilled to see the community contributing to improve `pop-cli.` 

If you need any help or have questions, please let us know here, or feel free to join our Telegram group.: https://t.me/Pop_Network

**Comment by brunopgalvao** [comment]
@chungquantin I am removing the label enhancement from this and adding discussion. I want the community to discuss whether this should be an added feature to pop-cli or not. If voted yes you can begin working on it.

I will also keep you in mind for other features.

cc: @AlexD10S

**Comment by chungquantin** [comment]
Personally, I think it should not pop up the browser automatically. After running `pop up parachain`, I would prefer another command asking me `if I want to open in a new window? Yes / No`

**Comment by brunopgalvao** [comment]
> Personally, I think it should not pop up the browser automatically. After running `pop up parachain`, I would prefer another command asking me `if I want to open in a new window? Yes / No`

The interactivity makes sense to me. This feature would be specific to the case where the user specifies the pop-node e.g. `pop up parachain -f ./network.toml -p https://github.com/r0gue-io/pop-node`



> I like the idea of opening a browser with Polkadot-Js for the local node. Maybe a `--no-browser` or `--no-popups` should go alongside this feature for users who may not want this behaviour.

These option could be helpful if the user wants to bypass the interactive question.

@AlexD10S any thoughts?

**Comment by pepoviola** [comment]
Hi 👋, first let me say that the cli looks awesome! 
The network instance you should have the info required to connect the node (exposed as ws_uri) but if you need only the port we can add a new method in the NetworkNode.

Thanks!!

**Comment by evilrobot-01** [comment]
Just a general FYI: we want to add an additional `--init` type option to this command which allows some other command to optionally be run after the network is spawned and parachain available. 

The initial use case is funding the dev accounts on PopNet via Xcm from relay (PopNet uses relay token) but this could be used by anyone to run any additional command/script as required.

Keeping this in mind might be useful in considering the eventual UX selected.

**Comment by chungquantin** [comment]
I found a CLI tool called Substrate Manager: https://github.com/omerdn1/substrate-manager which build the interactive flow very clean. Put it here as a reference.
